index.html

```diff
@@ -4,9 +4,65 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>First Portfoloio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;700&display=swap" rel="stylesheet">
 </head>
 <body>
-    
+  <header class="header">
+    <div class="left">
+      <a href="#">
+        <div class="logo">
+            My logo
+        </div>
+    </a>
+    </div>
+    <nav class="right">
+        <div class="menubar">
+            <div class="menubar1"></div>
+            <div class="menubar2"></div>
+            <div class="menubar3"></div>
+        </div>
+    </nav>
+</header>
```

Comment:
Kindly I suggest you follow the Figma template to make your design match the requirement, like margin, padding, font size, and font weight. Please go through this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=jtuuHJ5V6aki4hPg-0) to follow the Figma template.

> ### The Figma design
>
><img width="312" alt="Screenshot 2023-05-01 at 07 57 01" src="https://user-images.githubusercontent.com/91237525/235449305-f37edb7d-a0aa-4dbb-88ce-53023e02680e.png">
>
> ### Your design
>
><img width="372" alt="Screenshot 2023-05-01 at 07 56 21" src="https://user-images.githubusercontent.com/91237525/235449355-6bc122b1-ac83-49e5-8b31-709ccac14651.png">

Reply:
Fixed.